patterns/2-structured/maturity-model.md

```diff
@@ -0,0 +1,231 @@
+* SP-3:	There are rules and regulations to formalize the support on the product, given by a mature community.
+
+**Culture**
+
+There are multiple levels moving towards a collaborative culture.
+
+* CL-0: Silos - teams work independently but also in isolation.
+* CL-1: Reactive - teams work independently, but know how to react to flaws in dependencies.
+* CL-2: Contributive - teams actively help improve their dependencies by contributing.
+* CL-3: Activist - teams actively seek help, mentor and recruit new contributors.
+
+**InnerSource Roles**
+
+InnerSource comes with explicit roles. While in early stages some patterns may be useable without adopting those roles, communicating withing projects using explicit role titles becomes easier.
+
+* RO-0:	There are no specific roles helping InnerSource adoption. Only common development roles are present: developer, analyst, tester, etc.
+* RO-1: Occasionally some individuals and teams contribute to other projects. These are technical contributions, where the user/contributor role is seen. For some teams, it can be identified at least one member being a technical reference, who explains the development process to other development team members. He/she could be a candidate for covering the trusted commiter role.
+* RO-2:	An InnerSource Officer role is in charge for governance and support, including processes, etc. Identifies the education needs and ensures it is provided to the organization. Leads and mentors the organization in the engagement in IS projects. Is the first formal step in the way, defining the IS vision and roadmap. The organization has defined a trusted committer role, being a point of contact/reference not only for dev team members but also for external contributors. There is a standard process describing how to contribute to the community, contributor role is present. Data Scientist role is in charge of managing the traces of activity left by the InnerSource initiative, needed to measure the IS evolution. 	Trusted committer role will evolve to a more technical profile, and a community manager will be in charge of "energyze" the community, being his main responsibility to attract and retain new developers/users (contributors/community members).
+* RO-3: Evangelists are moving inside organization, to let others know about the current work, what InnerSource does and how to do it, and help others to understand and become part of the initiative. Non technical contributors appear.
+
+
+## Resulting Context
+
+All teams are aware of available best practices.
+
+Teams understand their level of InnerSource adoption.
+
+Prior to adopting InnerSource as a working model, teams are aware of the
+practices that are expected of them - both in the short term as well as in the
+long term.
+
+## Known Instances
+
+Entelgy, Zylk, Bitergia
```

Comment:
Hoping to make it more clear that these are 3 separate instances.
With that we might be able to move this pattern to the Validated (i.e. the highest pattern maturity) in a future PR.

```suggestion
* Entelgy
* Zylk
* Bitergia
```